### ISSUE TITLE
Shadowlings can Shadowwalk on moving space

### ISSUE THREAD

**Issue by ArcLumin** [HippieStationArchive/HippieStation13-1#2949]
Title. Pretty self explanitory

**Comment by trifornt-tinfornt** [comment]
If you fix this then you are a heartless monster.

even then only way they can really abuse it is by being an ascendant any way, you've already won the game

**Comment by ArcLumin** [comment]
Nah, not exactly. A dedicated shadowling with lots of metal and healing items could theoretically get to centcom without dying from spacelight

But centcom itself is maxed out with light you can't remove, soooo....

**Comment by trifornt-tinfornt** [comment]
Well then again, ignoring space light you'd still be taking constant damage from space pressure, the cold, and a lack of oxygen. And while healing items would help, the second you get ejected onto moving space you'll get sent back to a random Z-Level.
Assuming you do manage to escape fully and make your way to centcom, as you said, there's no light that can be removed and there isn't really anything there (besides maybe the chainsword?) that is salvageable and useful as a shadowling.
Good luck making it back, by the way.

regardless shadowlings are so insanely overpowered that I honestly think the effort of travelling to centcom would be a massive waste of time, even if you did manage to bring back a pulse rifle or seven. By the time you even get near the space station, and that's only if you aren't dead, your buddies have likely ascended.

Not really a priority bug and it's honestly a funny one, especially with ascendant lings fighting admins to the death.

**Comment by ArcLumin** [comment]
Shadowling gear makes you immune to space, and shadowlings are nobreath. The only issue is the spacelight, which is mitigated by hiding in lockers till your stuff recharges, a jetpack, and/or healing items.

As for moving space, only a tiny amount is on Z2, which is easily passed by if you know where to go. The ferry shuttle doesn't need any access to operate on the main console, so you can load it up with the entire armory (as well as nuke op gear and eswords and shit) and bring it back to all your fellow slings or hell, bring your thralls to centcom. Easy return to the station.

**Comment by ArcLumin** [comment]
Meanwhile, the bluespace artillery likewise requires no access. And once your thralls are there, they can use a rapid pipe dispenser to get past the immovable doors and walls. You can have your thralls operate centcom entirely

**Comment by trifornt-tinfornt** [comment]
That'd honestly be amazing. Best bug. Don't fix.
I'd still like to argue that Shadowlings are so overpowered that if you're remotely competent by the time a sling puts in the effort to get all of that stuff and attempt reaching centcom, once he even tries to get back his buddies have already ascended and that en devour was pointless.

**Comment by ArcLumin** [comment]
But yeah, I totally don't want to fix centcom arrival bugs. They're like the holy grail. It should be super difficult or even rng dependent, but I definitely want to get there even less than 1% of the time.

**Comment by blobbernaut** [comment]
it reminds me of that world -1 in mario, or some sort of hidden easter egg.

**Comment by ArcLumin** [comment]
Yeah, a bug to get to an area that you're not supposed to be, just like World 36-1. I love it
